xbmc/pvr/dialogs/GUIDialogPVRGuideControls.cpp

```diff
@@ -0,0 +1,17 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "GUIDialogPVRGuideControls.h"
+
+CGUIDialogPVRGuideControls::CGUIDialogPVRGuideControls()
+: CGUIDialog(WINDOW_DIALOG_PVR_GUIDE_CONTROLS, "DialogPVRGuideControls.xml")
```

Comment:
clang-format is currently set to add two spaces before `:`

Reply:
fixed